app/models/risk_condition.rb

```diff
@@ -93,10 +95,26 @@ def self.update_current_for_course(course_name, params)
       # else
       #   puts "case 5: previous conditions and current conditions match and no update is needed"
       end
+      WatchlistInstance.refresh_instances_for_course(course_name)
```

Comment:
I wonder if there is a better way to this as compared to having it at three different areas?

Reply:
But I think as of now this is okay.

Reply:
Yeah I was thinking about that too. But this was the fastest fix. We could add optimize this later PRs. I'll take a note.